demos/text_detection_demo/cpp/README.md

```diff
@@ -34,7 +34,7 @@ Options:
     -limit "<num>"             Optional. Number of frames to store in output. If 0 is set, all frames are stored.
     -m_td "<path>"               Required. Path to the Text Detection model (.xml) file.
     -m_tr "<path>"               Required. Path to the Text Recognition model (.xml) file.
-    -dt "<path>"               Required. Type of the decoder, either 'simple' for SimpleDecoder or 'ctc' for CTC greedy and CTC beam search decoders.
+    -dt "<type>"               Required. Type of the decoder, either 'simple' for SimpleDecoder or 'ctc' for CTC greedy and CTC beam search decoders.
```

Comment:
please update help message. You define default value, so it is not required anymore